src/main/java/com/google/api/codegen/ruby/RubyGapicContext.java

```diff
@@ -330,6 +330,10 @@ public String rubyTypeName(TypeRef typeRef) {
     return modules;
   }
 
+  public Iterable<String> getApiModules() {
+    return Splitter.on("::").splitToList(getApiConfig().getPackageName());
```

Comment:
Quickly reviewed `CommonGapicCodePathMapper`, but I am not sure it's worthwhile to extract the common pattern in this case, because the code is significantly different (it scans differently, and it needs to invoke toLowerCase).
Also having that in `CommonGapicCodePathMapper` makes me a bit nervous because it's unclear what that means in other languages.

I could extract `RubyCodePathMapper` class, but that sounds like another duplicated code among PathMappers -- and here is kinda 'one-liner' code, I think it's not worthwhile to creating a new path mapper class.